### ISSUE TITLE
error trying to view WSO data

### ISSUE THREAD

**Issue by tansaku** [CraftAcademy/visualizer#13]
![](https://www.dropbox.com/s/f1dl0agxty05j59/Screenshot%202016-11-28%2010.28.17.png?dl=1)

clicked all WSO data in the above and got the below:

![error ](https://www.dropbox.com/s/qbtdjn8pq4i7hle/Screenshot%202016-11-28%2010.27.30.png?dl=1)

**Comment by tochman** [comment]
Yes, I could replicate thist. It's the beginning of the week and some metrics are not available yet (I was coding this at the end of last week so I did not encounter this problem) Thx for reporting it.

**Comment by tochman** [comment]
@tansaku  The tmp fix for this issue is to generate report with previous weeks data - that has been implemented and deployed. As a long term fix we will be adding a fallback for each chart if there's no data available.
